src/test/groovy/nebula/plugin/publishing/maven/MavenBasePublishPluginIntegrationSpec.groovy

```diff
@@ -164,38 +164,30 @@ class MavenBasePublishPluginIntegrationSpec extends IntegrationTestKitSpec {
             apply plugin: 'java'
         '''.stripIndent()
 
-        String expectedPom = '''\
-            <?xml version="1.0" encoding="UTF-8"?>
-            <project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
-                xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
-              <modelVersion>4.0.0</modelVersion>
-              <groupId>test.nebula</groupId>
-              <artifactId>maventest</artifactId>
-              <version>0.1.0</version>
-              <name>maventest</name>
-            </project>
-        '''.stripIndent()
-
         when:
         runTasks('publishNebulaPublicationToTestLocalRepository')
 
         then:
-        new File(publishDir, 'maventest-0.1.0.pom').text == expectedPom
+        def resultPom = new File(publishDir, 'maventest-0.1.0.pom').text
```

Comment:
Gradle 6 adds marker info so I'm trying to find common ground between Gradle 5 and 6

Reply:
yeah, string comparison won't work unless gradle metadata is disabled but this is safer.

Reply:
+1 makes sense